PcdcAnalysisTools/blueprint/routes/views/survival/__init__.py

```diff
@@ -16,31 +16,47 @@
 @auth.authorize_for_analysis("access")
 def get_result():
     args = utils.parse.parse_request_json()
-    data = (
-        fetch_data(args)
-        if flask.current_app.config.get("IS_SURVIVAL_USING_GUPPY", True)
-        else fetch_fake_data(args)
-    )
-    return flask.jsonify(get_survival_result(data, args))
-
 
-def fetch_data(args):
     # TODO add json payload control
     # TODO add check on payload nulls and stuff
     # TODO add path in the config file or ENV variable
     _filter = args.get("filter")
+    filters = json.loads(json.dumps(_filter))
```

Comment:
@grugna Why do we need to dump/load JSON here? Isn't this effectively getting back the same object?